wowchemy/assets/scss/wowchemy/_footer.scss

```diff
@@ -43,3 +43,27 @@ footer .powered-by {
   height: 22px;
   vertical-align: text-bottom;
 }
+
+// Footer network icons //
+
+ul.footer-icon {
```

Comment:
Do we really need to duplicate styles which already exist, increasing the CSS payload?

Reply:
I guess not! The `icon-network` class is contained in an inline flex container, which doesn't center the icons on the page. I've wrapped in a `<div class="d-flex justify-content-center">`, which works.